frontend/src/app/components/about/about.component.html

```diff
@@ -299,9 +299,9 @@ <h3 i18n="about.community-integrations">Community Integrations</h3>
         <img class="image" src="/resources/profile/blixt.png" />
         <span>Blixt</span>
       </a>
-      <a href="https://github.com/ZeusLN/zeus" target="_blank" title="Zeus">
-        <img class="image" src="/resources/profile/zeus.png" />
-        <span>Zeus</span>
+      <a href="https://github.com/ZeusLN/zeus" target="_blank" title="ZEUS">
+        <img class="image" src="/resources/profile/ZEUS.png" />
```

Comment:
`ZEUS.png` in big letters when the image is still `zeus.png`

Reply:
I also changed the image name to `ZEUS.png`; I don't know how it's still showing like before. No worries, I'll change it back to `zeus.png.` We just needed to change the displayed name and logo.